compute_tools/src/compute.rs

```diff
@@ -669,20 +679,6 @@ impl ComputeNode {
         let config_time = Utc::now();
         if pspec.spec.mode == ComputeMode::Primary {
             self.configure_as_primary(&compute_state)?;
-            let conf = self.get_conn_conf(None);
-            tokio::task::spawn_blocking(|| {
-                let res = get_installed_extensions(conf);
```

Comment:
Hm, looks like this part got removed completely, but we need this for the extensions' metrics

Reply:
Oh, that's done in configure_as_primary() now. That was actually a bug in the previous refactoring PR (#11007). It duplicated that piece of code, and this removes the duplication again.

On second thoughts, I think this is a better place than configure_as_primary() for this, so I'll restore this and remove the duplicate from configure_as_primary()